chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx

```diff
@@ -278,7 +259,7 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
 
         // End chat
         BroadcastService.getMessageByEventName(BroadcastEvent.InitiateEndChat).subscribe(async () => {
-            if (state.appStates.skipChatButtonRendering !== true) {
+            if (state.appStates.hideStartChatButton === false) {
```

Comment:
if (!state.appStates.hideStartChatButton) {

Reply:
Same reason we need `Strict Equality` check here.